activerecord/test/cases/timestamp_test.rb

```diff
@@ -425,6 +425,17 @@ def test_all_timestamp_attributes_in_model
     toy = Toy.first
     assert_equal [:created_at, :updated_at], toy.send(:all_timestamp_attributes_in_model)
   end
+
+  def test_index_is_created_for_both_timestamps
+    connection = ActiveRecord::Base.connection
+
+    if connection.respond_to?(:indexes)
+      indexes = connection.indexes("flowers")
+      assert_equal ["created_at", "updated_at"], indexes.flat_map(&:columns).sort
+    else
+      warn "#{connection.class} does not respond to #indexes"
```

Comment:
This `warn` doesn't sound right. Either we expect all connections to support `indexes`, and we should just assume it's there, or it's fine for a connection to not, in which case we should quietly skip. (I don't happen to know which of those is true for this particular question, but I'm sure you can find some guiding precedent.)